samples/TagHelperSample.Web/Areas/Admin/Views/Admin/Index.cshtml

```diff
@@ -0,0 +1,7 @@
+﻿@*
+    For more information on enabling MVC for empty projects, visit http://go.microsoft.com/fwlink/?LinkID=397860
+*@
+@{
+}
+
+<a asp-action="Index" asp-controller="Home" asp-area="">Go home!</a>
```

Comment:
@javiercn and I talked about this API and both agree it's a littttle wonky (but works). @Eilon / @dougbu can either of you think of something better for navigating outside of an area from inside one?